Packs/VaronisSaaS/ReleaseNotes/1_0_4.md

```diff
@@ -0,0 +1,7 @@
+#### Integrations
+##### Varonis SaaS
+- Added varonis-integration header to all of the API calls
+- Multi-value parameters for commands now have pipe separator (|) instead of comma.
+- Varonis SaaS API v9 support.
+- Fixed bug with Close Reason enum mismatch.
+- Updated the Docker image to: *demisto/python3:3.10.13.89009*.
```

Comment:
```suggestion
- Added support for varonis-integration header to all API calls.
- Improved implementation of multi-value parameters for commands, now using pipe separator (|) instead of comma.
- Added support for Varonis SaaS API v9.
- Fixed an issue with Close Reason enum mismatch.
- Updated the Docker image to: *demisto/python3:3.10.13.89009*.
```